llarp/link/link_manager.cpp

```diff
@@ -398,6 +383,27 @@ namespace llarp
     });
   }
 
+  void
+  LinkManager::gossip_rc(const RemoteRC& rc)
+  {
+    for (auto& [rid, conn] : ep.conns)
+      send_control_message(rid, "gossip_rc", std::string{rc.view()}, nullptr);
```

Comment:
Minor:
Doesn't the third parameter default to `nullptr`?

Reply:
It does, yeah.  I'm contemplating whether we want it to default, actually.  The reason I explicitly pass `nullptr` here is to signal to future-me (or others reading it) that I intentionally don't have a response handler, rather than accidentally forgot.